### PR TITLE
Feature 2

### PR DESCRIPTION
**Pull request by abenaTenkorang**
**In this pull request,**

- We play with refactoring it.
- Added classes to our js file. 
- Adhered to the layout presented in the wireframe.

### CODE REVIEW

**Comment by DammyShittu** [comment]
Hi @abenaTenkorang,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Since this is a collaborative project, kindly ensure your coding partner contributes to this project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by abenaTenkorang** [comment]
Please my partner is not available, I have filed a complaint

File: index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/* eslint-disable lines-between-class-members */
+/* eslint-disable no-undef */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-unused-vars */
+/* eslint-disable max-classes-per-file */
```
Comment (DammyShittu) [inline index.js:5]: - [ ] The purpose of linters is to write maintainable code and maintain a standard across teams. Disabling linters defeats that purpose. Kindly remove these disabling and solve errors that come up.

  If you are stuck you can use google to find the solution for the errors. For example, the last error `max-classes-per-file`, requires that only one class should be in a file - which is a convention. You can separate classes into different files and export them, then import them accordingly.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @abenaTenkorang,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (abenaTenkorang) [reply index.js:5]: Can you please help me with that? @DammyShittu

Comment (abenaTenkorang) [reply index.js:5]: Please it is in the requirement that we can disable these linters.
day2

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand:  @abenaTenkorang, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :green_circle: 

- Descriptive PR :+1: 
- No Linter error :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**